### DISCUSSION TITLE
Unable to login to Azuracast

### DISCUSSION THREAD

**Discussion by gradation0746** [AzuraCast/AzuraCast#6255] [Q&A] [answered]
I initially set up Azuracast yesterday (it's incredible, btw).

I believe I enabled the option to forward to a page if the user isn't logged in (I can't remember exactly how the option was worded). I logged out and went back to continue set up today; however as I wasn't logged in, it keeps forwarding me to that page without the option to log in.

How can I remedy this so I can log in? Thanks in advance.

**Comment by Vaalyn** [comment, marked as answer]
Let's say you are running AzuraCast at `https://example.com`

When you set a `Homepage Redirect URL` in the Custom Branding page only the main `https://example.com` page will be redirected.
To login just visit `https://example.com/login` directly.

**Reply by gradation0746** [reply]
Hey thanks for your help. I had set the Homepage Redirect URL.
